app/models/abuse_report.rb

```diff
@@ -123,10 +125,27 @@ def send_report
     attach_work_download(ticket_id)
   end
 
+  def creator_ids
+    work_id = reported_work_id
+    return unless work_id
+
+    work = Work.find_by(id: work_id)
+    return "deletedwork" unless work
+
+    ids = work.pseuds.pluck(:user_id).push(*work.original_creators.pluck(:user_id)).uniq.sort!
```

Comment:
Could we use `sort` here instead of `sort!`, since the return value is being used (and we don't necessarily need to mutate the original)?